README.md

```diff
@@ -36,55 +36,90 @@ class TinySpacenetExperimentSet(rv.ExperimentSet):
+                                target_classes=[1],
+                                target_count_threshold=1000) \
                             .build()
 
-        backend = rv.BackendConfig.builder(rv.TF_OBJECT_DETECTION) \
+        # ------------- BACKEND -------------
+
+        backend = rv.BackendConfig.builder(rv.TF_DEEPLAB) \
                                   .with_task(task) \
                                   .with_debug(True) \
                                   .with_batch_size(8) \
                                   .with_num_steps(5) \
-                                  .with_model_defaults(rv.SSD_MOBILENET_V2_COCO)  \
+                                  .with_model_defaults(rv.MOBILENET_V2)  \
                                   .build()
 
+        # ------------- TRAINING -------------
+
         train_raster_source = rv.RasterSourceConfig.builder(rv.GEOTIFF_SOURCE) \
-                                                   .with_uri(train_image_uri) \
-                                                   .with_stats_transformer() \
-                                                   .build()
+                                             .with_uri(train_image_uri) \
+                                             .with_channel_order(channel_order) \
+                                             .with_stats_transformer() \
+                                             .build()
+
+        vector_source = train_label_uri
+        train_label_raster_source = rv.RasterSourceConfig.builder(rv.RASTERIZED_SOURCE) \
+                                                         .with_vector_source(vector_source) \
+                                                         .with_rasterizer_options(2) \
+                                                         .build()
+        train_label_source = rv.LabelSourceConfig.builder(rv.SEMANTIC_SEGMENTATION) \
+                                                 .with_raster_source(train_label_raster_source) \
+                                                 .build()
 
         train_scene =  rv.SceneConfig.builder() \
                                      .with_task(task) \
                                      .with_id('train_scene') \
                                      .with_raster_source(train_raster_source) \
-                                     .with_label_source(train_label_uri) \
+                                     .with_label_source(train_label_source) \
                                      .build()
 
+        # ------------- VALIDATION -------------
+
         val_raster_source = rv.RasterSourceConfig.builder(rv.GEOTIFF_SOURCE) \
                                                  .with_uri(val_image_uri) \
+                                                 .with_channel_order(channel_order) \
                                                  .with_stats_transformer() \
                                                  .build()
 
+        vector_source = val_label_uri
```

Comment:
This local variable doesn't seem to aid comprehension and makes the example longer.

Reply:
Same for previous setting of vector_source

Reply:
Done.